src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,59 @@
+.icon {
+  position: relative;
+  display: block;
+  width: 100%;
```

Comment:
display block has width 100% be default

Reply:
yes, but if I remove it, the width becomes 0 and the icon is not displayed %)

Reply:
Ok, I saw. It's strange for me. If I remove display block property - nothing is changing. Maybe, you can remove it😅

Reply:
I think it's because of grid parent